docs/technical-details/release-notes/4.tucker/README.md

```diff
@@ -2,13 +2,15 @@
 
 HarperDB version 4 ([Tucker release](./tucker.md)) represents major step forward in database technology. This release line has ground-breaking architectural advancements including:
 
-## [4.0](./4.0.0.md)
-* New clustering technology that delivers robust, resilient and high-performance replication
-* Major storage improvements with highly-efficient adaptive-structure modified MessagePack format, with on-demand deserialization capabilities
+## [4.4](./4.4.0.md)
+* Native replication (codename "Plexus") which is faster, more efficient, secure, and reliable than the previous replication system and provides provisional sharding capabilities with a foundation for the future
+* Computed properties that allow applications to define properties that are computed from other properties, allowing for composite properties that are calculated from other data stored in records without requiring actual storage of the computed value
+* Custom indexing including composite, full-text indexing, and vector indexing
 
-## [4.1](./4.1.0.md)
-* New streaming iterators mechanism that allows query results to be delivered to clients _while_ querying results are being processed, for incredibly fast time-to-first-byte and concurrent processing/delivery
-* New thread-based concurrency model for more efficient resource usage
+## [4.3](./4.3.0.md)
+* Relationships, joins, and broad new querying capabilities for complex and nested conditions, sorting, joining, and selecting with significant query optimizations
```

Comment:
@jcohen-hdb @kriszyp 

What are your thoughts on rephrasing to what I have below?

Enhanced Querying and Joins: New support for relationships and joins, along with expanded query capabilities for handling complex, nested conditions. Sorting, joining, and selecting have been optimized for better performance, making complex queries faster and more efficient.

Advanced Transaction Support: Improved handling of CRDTs and added support for large integers (BigInt), enabling more robust and precise data operations.

Improved Management Tools: A revamped local studio and enhanced CLI with new features for easier management and smoother workflows.